alembic/versions/2020081018_added_group_uuid_to_worker_9d3ab0b9d304.py

```diff
@@ -6,13 +6,13 @@
 
```

Comment:
Is there a way to hook up alembic with flake8 so we don't have to manually make these modifications?

Reply:
I asked the question in the channel. Needs some further investigation.

Reply:
Can you create an issue for it for now?

Reply:
sure.

Reply:
https://github.com/codalab/codalab-worksheets/issues/2896